Validation function (#27)
cytodata/single-cell-classifier: 3.train_ebimage/1.random_forest/README.md

- warm_start=False
- class_weight=None

### Experiment setup

> To find out how stable this model come to a result the script is run. 

Reviewer: ```suggestion
We run a script to determine model performance and model stability. 
```

Keeping in first person

Every time the script is run another results in added to the list. 
These results show what the performance is (the mean) and how stable it get there (the variance).

> This can be used to compare different models with each other and conclude if a model is a better fit or not.  

Reviewer: ```suggestion
This can be used to compare different models with each other and conclude if a they have different stability. 
```

"Fit" is a specific word that could be confusing to some

### Results

> The balanced f1-score on the validation set is [0.328 ± 0.0030 with n=6](results/all_scores.xlsx).

Reviewer: ```suggestion
The balanced f1-score on the validation set is [0.328 ± 0.0030 with 6 model initializations](results/all_scores.xlsx).
```

this is great to have a range!! nice work. I change n=6 since n usually refers to sample size and (i think) we're referring to 6 model initializations instead.

> This is very low, but what is interesting is the [confusion matrix](results/0/confusion_matrix.png).

Reviewer: ```suggestion
This is very low, but what is interesting is the [confusion matrix](results/0/confusion_matrix.png) (shown is one example initialization).
```

Adding a note that one example initialization is shown
